### PR TITLE
Build filtering parameter documentation

### PR DESCRIPTION
**Pull request by slapec93**
Trello card:
https://trello.com/c/x4MZ4rgv/1775-write-api-docs-for-build-list-queries

### CODE REVIEW

**Comment by slapec93** [comment]
[Write API docs for build list queries](https://trello.com/c/x4MZ4rgv/1775-write-api-docs-for-build-list-queries)

**Comment by slapec93** [comment]
[Write API docs for build list queries](https://trello.com/c/x4MZ4rgv/1775-write-api-docs-for-build-list-queries)

File: docs/api/v0.1.md [hunk 1]
```diff
@@ -318,6 +324,686 @@ curl -H 'Authorization: token THE-ACCESS-TOKEN' 'https://api.bitrise.io/v0.1/app
...
+  - *`push`*: when the response contains only the builds which were triggered by a commit push
+  - *`pull-request`*: when the response contains only the builds which were triggered by a pull request
+- **`after`** (integer): Giving this parameter, in the response there will be only those builds which were triggered after or at the given time. The value has to be passed as UNIX timestamp.
+- **`before`** (integer): Similar to the previous one this filters by the time when the build was triggered, but this retrieves the ones before or at the given time. The value has to be passed as UNIX timestamp.
+- **`workflow`** (string): The response will contain only the builds triggered with the specified workflow.
+
+##### Example `curl` requests
+
+Request with `status` filtering (aborted builds):
+```
```
Comment (viktorbenei) [inline docs/api/v0.1.md]: add an empty line before this

File: docs/api/v0.1.md [hunk 2]
```diff
@@ -318,6 +324,686 @@ curl -H 'Authorization: token THE-ACCESS-TOKEN' 'https://api.bitrise.io/v0.1/app
...
+curl -H 'Authorization: token THE-ACCESS-TOKEN' 'https://api.bitrise.io/v0.1/apps/APP-SLUG/builds?status=3'
+```
+
+Response:
+```
```
Comment (viktorbenei) [inline docs/api/v0.1.md]: add empty line before this line

File: docs/api/v0.1.md [hunk 3]
```diff
@@ -318,6 +324,686 @@ curl -H 'Authorization: token THE-ACCESS-TOKEN' 'https://api.bitrise.io/v0.1/app
...
+  ],
+  "paging": {
+    "page_item_limit": 50,
+    "total_item_count": 19
+  }
+}
+```
+
+Request with `branch` filtering (builds on _develop_ branch):
+```
```
Comment (viktorbenei) [inline docs/api/v0.1.md]: add empty line before this line

File: docs/api/v0.1.md [hunk 4]
```diff
@@ -318,6 +324,686 @@ curl -H 'Authorization: token THE-ACCESS-TOKEN' 'https://api.bitrise.io/v0.1/app
...
+curl -H 'Authorization: token THE-ACCESS-TOKEN' 'https://api.bitrise.io/v0.1/apps/APP-SLUG/builds?branch=develop'
+```
+
+Response:
+```
```
Comment (viktorbenei) [inline docs/api/v0.1.md]: add empty line before this line

File: docs/api/v0.1.md.gg [hunk 5]
```diff
@@ -175,15 +175,91 @@ Build times:
...
+  - *`push`*: when the response contains only the builds which were triggered by a commit push
+  - *`pull-request`*: when the response contains only the builds which were triggered by a pull request
+- **`after`** (integer): Giving this parameter, in the response there will be only those builds which were triggered after or at the given time. The value has to be passed as UNIX timestamp.
+- **`before`** (integer): Similar to the previous one this filters by the time when the build was triggered, but this retrieves the ones before or at the given time. The value has to be passed as UNIX timestamp.
+- **`workflow`** (string): The response will contain only the builds triggered with the specified workflow.
+
+##### Example `curl` requests
+
+Request with `status` filtering (aborted builds):
+```
```
Comment (viktorbenei) [inline docs/api/v0.1.md.gg]: add empty line before this line

**Comment by slapec93** [comment]
[Ability to filter pull request id as well](https://trello.com/c/qYo1VTkm/1920-ability-to-filter-pull-request-id-as-well)

**Pull request merged**